android/app/build.gradle

```diff
@@ -121,6 +121,9 @@ android {
         versionCode System.getenv('CIRCLE_BUILD_NUM') ? System.getenv('CIRCLE_BUILD_NUM').toInteger() : 1
         versionName "1.1." + (System.getenv('CIRCLE_BUILD_NUM') ? System.getenv('CIRCLE_BUILD_NUM') : "1")
         multiDexEnabled true
+        // The resConfigs attribute will remove all not required localized resources while building the application,
+        // including the localized resources from libraries.
+        resConfigs 'en', 'it'
```

Comment:
do we have any localized resources? I believe we deal with localization with react-i18n - @fpersico ?

Reply:
This concerns the localizations and resources of the **native libraries**